Gem/Code/Source/AutoGen/NetworkTeleportCompatibleComponent.AutoComponent.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+
+<Component
+    Name="NetworkTeleportCompatibleComponent"
+    Namespace="MultiplayerSample"
+    OverrideComponent="true" 
+    OverrideController="true" 
+    OverrideInclude="Source/Components/NetworkTeleportCompatibleComponent.h"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+	<ComponentRelation Constraint="Required" HasController="true" Name="NetworkTransformComponent" 
+                       Namespace="Multiplayer" Include="Multiplayer/Components/NetworkTransformComponent.h" />
+
+    <RemoteProcedure
+        Name="SendTeleportLocation" 
```

Comment:
Hmm, something strikes me SendTeleportLocation is the wrong name. The person handling this will teleport. Not sure the standard, but like OnTeleport or just Teleport? Then you have HandleTeleport() which reads nicely?

Reply:
Agree on this. My opinion is it's more instructive to describe what's happening to the entity rather than what the code is doing here. TriggerTeleport or Teleport could work here.

Reply:
yeah, I like "HandleTeleport" better. Will update

Reply:
Be careful that doesn't turn `HandleSendTeleportLocation` into `HandleHandleSendTeleportLocation`

Reply:
+1 I meant for the resulting handler func. I've been burned by the "HandleHandle..." before x(